docker-entrypoint.sh

```diff
@@ -29,16 +29,21 @@ if [ "$1" = 'postgres' ]; then
 		fi
 		
 		: ${POSTGRES_USER:=postgres}
-		if [ "$POSTGRES_USER" = 'postgres' ]; then
-			op='ALTER'
-		else
-			op='CREATE'
+		: ${POSTGRES_DB:=$POSTGRES_USER}
```

Comment:
Don't we get the default `postgres` db from the `gosu postgres initdb` line above? I think @rothgar alluded to that in one of the comments (the fact that the default `postgres` db would still exist).